javascript/node/selenium-webdriver/test/chrome/devtools_test.js

```diff
@@ -124,13 +125,23 @@ test.suite(
 
       server.start()
 
-      it.only('denies entry if username and password do not match', async function() {
-        // eslint-disable-next-line no-unused-vars
-        await driver.get('http://the-internet.herokuapp.com/basic_auth')
+      it('denies entry if username and password do not match', async function() {
+        const pageCdpConnection = await driver.createCDPConnection('page')
+
+        await driver.register('random', 'random', pageCdpConnection)
+        await driver.get(server.url() + '/protected')
+        let source = await driver.getPageSource()
+        assert.equal(source.includes('Access granted!'), false)
+        await server.stop()
```

Comment:
we can ignore stopping server here as the next spec stops the server ( i dont see any server started in next spec so we can use the existing server)

Reply:
Oh yes, thats correct. Removed on `server.stop`

Locally I tested using `.only` so that did no create any issue.

Fix pushed